### PR TITLE
[AMQ-8278] Add a dynamic news feed section to home page

### PR DESCRIPTION
**Pull request by lucastetreault**
Building on Matt's idea here: https://github.com/apache/activemq-website/pull/57 as discussed on slack. 

Sorry for the duplicate PR, I wasn't sure how to propose changes to the original one.

### CODE REVIEW

**Comment by gemmellr** [comment]
@jbertram 
> 4. The summary information (i.e. bugs, enhancements, dependencies) is no longer present. I thought this info was useful as it succinctly communicated the character of the release.

I think that was probably removed after I said on the other PR that I didnt see it is as being all that useful, especially for a feed area I dont think it remotely warrants the space burned on it. Even whats left is still too big IMO. 

I dont think it truly helps you usefully characterise the release most of the time personally. Without seeing an overview of what the changes are you cant really tell much of anything about their importance or relevance from the number of them. The number itself is also often just a consequence of how long it is since the last release was done, rather than anything inherent about the actual changes themselves (e.g Artemis 2.18.0 took 6 months, but many of those changes could/should have gone out months earlier with followup releases later for more changes, they just didnt. More than ditto for 5.17.x when it happens). 

Plus it complicates the steps needed to add a release to the site. People already seem to find that too much of a chore to do as it is so I would skip it for that alone.

File: src/_includes/news.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        {% continue %}
+    {% else %}
+        {% assign collections = collections | append:'|' | append:release.collection %}
+    {% endif %}
+    <div class="col-lg-4 col-md-auto">
+        <div class="card card-orange">
+            <div class="card-body">
+                {% if release.collection contains "artemis" %}
+                    <h5 class="card-title text-blue">Artemis {{release.version}} Release</h5>
+                    <a style="margin-top: 10px" href="./components/artemis/download/past_releases" class="btn btn-secondary float-right">Release<i class="fa-download fa btn-icon"></i></a>
```
Comment (gemmellr) [inline src/_includes/news.md:24]: This should link to the download page. The past/previous releases (as in not-current releases) page doesnt actually contain the release the entry is about.

**Comment by mattrpav** [comment]
> Here's my feed-back from #57 that wasn't addressed plus a few more comments:
> 
>     1. It's not clear to me why specifically this is needed. Links to both Classic and Artemis releases are available just a bit further down the page, and NMS and CMS release links are very clearly visible on their respective component pages.

The intent is to create a news feed. Releases, CVE, blogs, articles, roadmaps, etc.

>     2. In terms of screen real-estate and attention, this is a pretty significant change to the main page. I'm not sure it makes sense for this feed to occupy such a prominent position. Maybe a new top-level menu item should be added and all these details could go there.

The intent is to put the news feed front-and-center. The news feed reflects the community activity. There is always a ton of activity around ActiveMQ, but the project page is very stagnant.  

>     3. The summary information (i.e. bugs, enhancements, dependencies) is no longer present. I thought this info was useful as it succinctly communicated the character of the release.

+100 I agree with you on this . I think the counts convey the most important info about the release.

>     4. I don't think it's useful to have more than one release per component displayed. Otherwise we're advertising older releases which doesn't seem helpful and certainly not worth such a valuable position on the website.

I think the news feed should just be natural. If there is a hot-fix release right after one, that should be reflected. x.y.1 releasing 1 day after x.y.0 with 1 bug fix doesn't capture the reality that the release is changes from x.y.0 + x.y.1.  (Or CVE, etc)

>     5. This is not so much a "news feed" as it is a "release feed."

Agreed, this should be flipped back to a news feed, and pull just the top-3 items.

**Comment by mattrpav** [comment]
> @jbertram
> 
> > 1. The summary information (i.e. bugs, enhancements, dependencies) is no longer present. I thought this info was useful as it succinctly communicated the character of the release.
> 
> I think that was probably removed after I said on the other PR that I didnt see it is as being all that useful, especially for a feed area I dont think it remotely warrants the space burned on it. Even whats left is still too big IMO.
> 
> I dont think it truly helps you usefully characterise the release most of the time personally. Without seeing an overview of what the changes are you cant really tell much of anything about their importance or relevance from the number of them. The number itself is also often just a consequence of how long it is since the last release was done, rather than anything inherent about the actual changes themselves (e.g Artemis 2.18.0 took 6 months, but many of those changes could/should have gone out months earlier with followup releases later for more changes, they just didnt. More than ditto for 5.17.x when it happens).
> 
> Plus the 'number of bugs/etc' summary complicates the steps needed to add a release to the site. People already seem to find that too much of a chore to do as it is so I would skip it for that alone.

I agree w/ @jbertram and contend this approach is an "engineer" perspective vs an end-user that manages a handful of ActiveMQ clusters along with 5 or 6 other products. We should target the end-user audience with the community front-page.  Kafka and Camel are doing really well with this approach.

**Comment by gemmellr** [comment]
My 'approach' for not being interested in the counts is considered with my user hat on. That whenever I am looking at a new release of anything as a user of it, I really dont have any interest at all in how many changes are in it. I care _what_ changed that impacts me, which the counts tells me next to nothing about.

Releases with 1 change can be very important to me as a user, or completely irrelevant. Just like ones with 100 changes. Knowing the count doesnt help me determine that as it tells me essentially nothing by itself (except perhaps to infer whether the release is long overdue; though if it is I probably know that if I've been waiting on it). Only knowing _what_ changed gives me the detail I want as a user.

Neither Kafka or Camel have such counts taking up a huge chunk of real estate on their front page that I can see. Camel has it in a linked blog post admittedly; I find it equally useless there though.

I will cede the last part, caring about what is needed to update the website, is me as with my engineer hat on trying to make releases 'simple'..since they are apparently already considered enough effort that most people dont like doing (or completing) them in a timely fashion.

**Comment by lucastetreault** [comment]
Pushed a few more changes after chatting with @mattrpav.

News feed on the front page with the 3 latest news items and a short description where you can include information about the number of changes, etc. The short description is pulled from the front matter of the release or news article so it should be pretty low impact to the release process: 

```
---
version: 2.18.0
release_date: 2021-08-12
title: ActiveMQ Artemis 2.18.0
shortDescription: Includes 12 new features, 79 bug fixes, 19 dependency upgrades, and 46 enhancements.
# Docs subdir name for past-releases and previous-docs pages, 'latest' is always used on the main download page.
docs_version: 2.18.0
---
```

image
 
A news feed with all the releases and news: 
image

Dedicated release pages for Artemis releases so I have some place to link to from the front page: 
image

A new collection `_news` for CVEs, blogs, articles, roadmaps, etc. I included a sample but would probably remove this before merging: 
image

**Comment by jbertram** [comment]
> Plus the 'number of bugs/etc' summary complicates the steps needed to add a release to the site.

Although I do see value in having this summary, I think this is a compelling point. The less manual work needed for a release the better.

> The news feed reflects the community activity. 

So is the underlying goal here really just to help users quickly and easily see that there's activity within ActiveMQ? If so, is there a specific reason why this is a concern? Are users coming to you and asking if ActiveMQ is still active (no pun intended) or is this just a proactive step you think the project should take? Please elaborate.

For what it's worth, there are lots of ways to see project activity (e.g. Jira, GitHub commits, mailing lists, etc.) all of which are linked to from the website and readily accessible to interested parties.

> There is always a ton of activity around ActiveMQ, but the project page is very stagnant.

*Stagnant* is a bit pejorative, but from a design perspective I actually quite like a mostly static main page as I think it's good for usability. It certainly helps returning users who expect key touch points and information to be in the same place as before.

Currently the main page clearly states the purpose of the project, what platforms it supports, what protocols it supports, some basic use-cases, etc. All this information is especially important to new users. It's not clear to me that a demonstration of recent activity deserves to be more "front and center" than this.

I have no issue with a new top-level menu item called "News" (or whatever) that would have the content you're suggesting. I think such a page would follow a pretty well-established web paradigm. I also have no issue with some kind of "Latest News" widget on the main page with the 3 most recent news items, but I think it shouldn't take such a prominent position.

Given the not-so-distant past of the website with all the broken links, out-dated/irrelevant/repeated information, etc. I'm loathe to add much complexity. The simpler the website is the easier it will be to update (and potentially migrate) in the future. The deeper we get into Jekyll features the more work it will be to get out eventually. By this I don't mean we shouldn't have a news page; I just think we should proceed with caution to avoid repeating previous mistakes.

**Comment by gemmellr** [comment]
> 
> 
> Pushed a few more changes after chatting with @mattrpav.
> 
> News feed on the front page with the 3 latest news items and a short description where you can include information about the number of changes, etc. The short description is pulled from the front matter of the release or news article so it should be pretty low impact to the release process:
> 

I speak from a view of having pointed out on a frankly ridiculous number of occasions that people messed up making utterly trivial changes to update the website with a new release. You automated some bits originally to make it less effort, I thought that was great so I then made further improvements and did some others. Its obviously not hard, but from many prior showings people clearly dont want to go to the effort. Anything that makes it even slightly more 'difficult' like this is a bad idea. 

I wouldnt mind the extra step/detail being added here if I thought it actually added particularly useful info or made the site look nicer, but as I've said I personally dont think it does either. I still have to click to find out truly useful info, so the step and the space it takes up might as well not be there in my view.

> 
> image
> 

Repeating prior comment that the banner and news feed take up way too much space. More so now the latter has about doubled in size. That the release happened and when is all I think it should show. It can then link to areas that have more information if desired.

I guess from the description we are back to having multiple release entries for the same component, so I'll also repeat that I think that is a bad idea. Nothing so prominent as the front page should be linking to non-current non-mirrored releases.

> Dedicated release pages for Artemis releases so I have some place to link to from the front page:
> image
> 

I think it is better with a single download page being pointed to. People should always be directed to the latest release, particularly from any highly visible location such as the front page, as they are the only releases that are mirrored.

If people want an old non-current release, there is already a simple page for those (which is linked to right from the top of the download page) or they can hit the archives directly. For the new page, it is easy enough to tell for older entries whether a release is 'current' or not and adapt the stale entry link to go to the past releases page instead of the download page.

If we were to have prominent links pointing to individual release download pages then I think they should be set up to have a nice big warning emerge at the top noting if they are old releases and to go get the current release from the download page.

(I also dislike that e.g existing 5.x release announcements go to a speciifc page rather than its download page, as that way people are always directed there when coming across old announcements rather than to the latest release. They should really have the same warnings added)

**Comment by mattrpav** [comment]
> > Pushed a few more changes after chatting with @mattrpav.
> > News feed on the front page with the 3 latest news items and a short description where you can include information about the number of changes, etc. The short description is pulled from the front matter of the release or news article so it should be pretty low impact to the release process:
> 
> I speak from a view of having pointed out on a frankly ridiculous number of occasions that people messed up making utterly trivial changes to update the website with a new release. You automated some bits originally to make it less effort, I thought that was great so I then made further improvements and did some others. Its obviously not hard, but from many prior showings people clearly dont want to go to the effort. Anything that makes it even slightly more 'difficult' like this is a bad idea.
> 
> I wouldnt mind the extra step/detail being added here if I thought it actually added particularly useful info or made the site look nicer, but as I've said I personally dont think it does either. I still have to click to find out truly useful info, so the step and the space it takes up might as well not be there in my view.
> 
> > image
> 
> Repeating prior comment that the banner and news feed take up way too much space. More so now the latter has about doubled in size. That the release happened and when is all I think it should show. It can then link to areas that have more information if desired.
> 
> I guess from the description we are back to having multiple release entries for the same component, so I'll also repeat that I think that is a bad idea. Nothing so prominent as the front page should be linking to non-current non-mirrored releases.
> 
> > Dedicated release pages for Artemis releases so I have some place to link to from the front page:
> > image
> 
> I think it is better with a single download page being pointed to. People should always be directed to the latest release, particularly from any highly visible location such as the front page, as they are the only releases that are mirrored.
> 
> If people want an old non-current release, there is already a simple page for those (which is linked to right from the top of the download page) or they can hit the archives directly. For the new page, it is easy enough to tell for older entries whether a release is 'current' or not and adapt the stale entry link to go to the past releases page instead of the download page.
> 
> If we were to have prominent links pointing to individual release download pages then I think they should be set up to have a nice big warning emerge at the top noting if they are old releases and to go get the current release from the download page.
> 
> (I also dislike that e.g existing 5.x release announcements go to a speciifc page rather than its download page, as that way people are always directed there when coming across old announcements rather than to the latest release. They should really have the same warnings added)

**Comment by mattrpav** [comment]
> Given the not-so-distant past of the website with all the broken links, out-dated/irrelevant/repeated information, etc. I'm loathe to add much complexity. The simpler the website is the easier it will be to update (and potentially migrate) in the future. The deeper we get into Jekyll features the more work it will be to get out eventually. By this I don't mean we shouldn't have a news page; I just think we should proceed with caution to avoid repeating previous mistakes.

I contend that adding a single data file is easier and less error prone than editing a static page.

We can't use a web framework, such as Jekyll, and simultaneously not use it.  The approach here is to separate data from the component that renders the page. It is super minor. The data is in a text file-- you can't make a "supporting migration" any easier than this. 

News item targets: CVE's, Releases, Blogs, Articles, interesting SO, etc.

**Comment by mattrpav** [comment]
Proposal:

 - [ ] Update the past releases pages to be dynamic and pull from the 'release data'
 - [ ] Update the release data to add a field to indicate release stream (ie 5.16.x, 2.18.x, etc)
 - [ ] Update the download page to show most recent release for 'active' streams. This would just be a hard-coded values (ie 5.16.x, 5.17.x and 2.18.x, etc) in the page to keep the complexity low

**Comment by lucastetreault** [comment]
> Decisions?:
> 
> 1. Single news entry per-component

I don't have a strong opinion either way on this one... I think there's value in showing sequential releases... E.g.: a 2.18.0 release and 2.18.1 release 2 days later is interesting context for users. I understand the worry about driving users to versions we don't want them using though. 

> Proposed updates:
> 
> 1. Update the past releases pages to be dynamic and pull from the 'release data'
> 2. Update the release data to add a field to indicate release stream (ie 5.16.x, 2.18.x, etc)
> 3. Update the download page to show most recent release for 'active' streams. This would just be a hard-coded values (ie 5.16.x, 5.17.x and 2.18.x, etc) in the page to keep the complexity low
> 4. Add follow-on task to re-work the top-banner

1-3 are all already done!

**Comment by gemmellr** [comment]
> 
> 
> Decisions?:
> 
>     1. Single news entry per-component
> 
> 
> Proposed updates:
> 
>     1. Update the past releases pages to be dynamic and pull from the 'release data'
> 

It already does.

>     2. Update the release data to add a field to indicate release stream (ie 5.16.x, 2.18.x, etc)

There is already a 'current stream' version config, and the releases are sorted so its easy to determine the latest. 

All thats needed is to match to the relevent component to know which versions should be compared. So perhaps the releases metadata for each component should have a 'component' field.

> 
>     3. Update the download page to show most recent release for 'active' streams. This would just be a hard-coded values (ie 5.16.x, 5.17.x and 2.18.x, etc) in the page to keep the complexity low

This is already in place.

The downlaod page is already dynamic, based on the above 'current versions' config. It should however many streams you like: https://github.com/apache/activemq-website/blob/6d37695e9bd11cdbfbe0123e286ab7063d70d0c2/src/_data/current_releases.yml
> 
>     4. Add follow-on task to re-work the top-banner

As above this is part of this change IMO

**Comment by lucastetreault** [comment]
If we agree that a news feed is worthwhile - I don't think it can get much easier than this. We could drop the short description from the version release front matter so there's no extra work for releases but I think the news feed ends up too light in that case. In fact, the short description isn't necessary and could simply be omitted as is with the current implementation. 

Regarding a single vs. multiple releases per component in the front page feed. I see value in both... @gemmellr has pretty strong opinions and it's a pretty easy change one way or the other so I would be fine going back to a single release per component. It's not like we can't change it in the future if something changes. 

About the size, placement and design fo the front page news feed. I like the way it is now and I think it **should** be prominent. However, I am not a designer and could be convinced otherwise. If someone wants to mockup some changes or propose something I can definitely play with it :)

**Comment by jbertram** [comment]
> We can't use a web framework, such as Jekyll, and simultaneously not use it.

I apologize if my comment communicated this logical fallacy. I'm not saying we shouldn't use Jekyll. Clearly we are _already_ doing that. My point was simply that we should be circumspect in _how_ we use it, especially with more advanced features. If this specific change is really as simple as it's made out to be then I don't see a problem with regard to the new complexity. Still, the point is worth making to ensure we cover our bases especially given our unique context (i.e. a clear history of poor website maintenance).

Might I suggest something like the following?
Screenshot 2021-08-31 at 15-32-20 ActiveMQ
Noteworthy changes:

- "News" top-level menu item
- Main banner height slightly reduced to bring subsequent items up a bit
- Very clear and succinct bulleted list of most recent news items in chronological order
    - Decreasing clutter tends to increase readability therefore...
        - Items are simple links which can be followed for more details
        - Redundant "Read More" button is eliminated
    - The list is not immediately after the banner, but is still clearly displayed without scrolling; I felt this was a good compromise as I would personally prefer it be even further down
- A bit cleaner visual separation from the surrounding text items

Regardless of the visual design I agree with Robbie that linking to old releases from the main page should be avoided. I also don't like the page-per-release. I think that each component should have a "latest" page and a "past releases" page and that's it.

**Comment by mattrpav** [comment]
@jbertram thanks for taking the time to build a mock-up of your ideas. Picture-thousand words type deal right there.

I'm annotating and will share back.

**Comment by gemmellr** [comment]
The thing I absolutely against is having prominent entries on the front page as 'news', linking through to specific old release pages with non-mirrored downloads for the stale/old release. We should direct folks to the download page that has the latest release and if they are determined on using old non-mirrored ones they can find it easily from there given we link to the old releases page.

If we do that, then I would say having old release entries on would be far less of an issue, even though I still really dont think stale release entries should be there at all, especially instead of e.g putting continued focus on another still-current release of some other component, or a CVE notice, or really anything else considered news worthy enough to be there.

I probably prefer the horizontal approach, and think its nice being above the blurb but only if much much smaller. I do like the single-line entry size in Justins alternative idea (but would include the date), and especially getting rid of the 'Read More' button and all the space used by just making the entry title be the actual link. 

Anything done should perhaps have a small link to the actual full news page if there is to be one (again perhaps just using the title would do). 

E.g for me something simple like a single line running across with the link and date would be nicer, i.e:
>\<Heading Banner\>
>News:   ActiveMQ Bar X.Y.0  (Sept 3, 2021)   |  ActiveMQ Foo  X.Y.1  (Sept 2, 2021)  |  ActiveMQ Foo  X.Y.0 (Sept 1, 2021)
> \<Main Blurb\>

Its still going to be very prominent just because of where it is and it being the first content of meaning on the page. It doesnt need to take up like 20% of the visible space on a screen.

**Comment by gemmellr** [comment]
That was my mock up.

**Comment by jbertram** [comment]
@gemmellr, something like this?

Screenshot 2021-09-01 at 09-49-06 ActiveMQ

I do like the horizontal orientation here, but was concerned that the date was going to make it crowded so I changed to the format (i.e. YYYY-MM-DD) and removed "ActiveMQ" from each of the titles since that's redundant.

**Comment by mattrpav** [comment]
My attempt at merging the mock concepts and compromising on ideas.

Having all the text as linkable is a dated UI/UX concept. We can merge the ideas on saving space and still providing the link to the news item by using a "..more" linkable text. Camel uses "Read more", so this would be more compressed, while aligning to precedent.

Keys:
- [ ] Reduce vertical space 50% or more
- [ ] Drop 'Read More' button and convert to a single "...more" clickable link text instead of button (see: Camel website)
- [ ] Drop the 'News' heading.. redundant and saves space

2021-activemq-main-ltetreailt

**Comment by gemmellr** [comment]
> @gemmellr, something like this?
> 
> \<snipped pic\>
> I do like the horizontal orientation here, but was concerned that the date was going to make it crowded so I changed to the format (i.e. YYYY-MM-DD) and removed "ActiveMQ" from each of the titles since that's redundant.

Pretty damn close yep. Perhaps a tiny smidgen bigger and dropping the seperator, or perhaps just make the seperator a more subtle colour. 

I was envisaging the date wouldn't necessarily be part of the link, but perhaps a smaller or a different colour (e.g as in earlier attempts) which i think could make it seem less crowded. I probably prefer the textual date rather than YYYY-MM-DD, mainly since MM-DD grates a bit ;)

**Comment by gemmellr** [comment]
> My attempt at merging the mock concepts and compromising on ideas.
> 
> Having all the text as linkable is a dated UI/UX concept. We can merge the ideas on saving space and still providing the link to the news item by using a "..more" linkable text. Camel uses "Read more", so this would be more compressed, while aligning to precedent.
> 

I actually think the button or 'More..' style text is a little awkward or out of place in these cases when there is essentially nothing to read/do except go get the release. I'm not a particular fan of the camel equivalent but I would say their entries are often quite different through linking to a small encyclopedia, or e.g not including any download details at all for the release related entries, whereas thats all I would link to here. 

This far I most like the recent effort from Justin. Perhaps with some of the colouring from the others.

**Comment by mattrpav** [comment]
My css is failing me.. I'm sure @lucastetreault can sort it out. The final rev should be reduced by over 50%, but hopefully conveys the key elements from @gemmellr @jbertram ideas.

2021-activemq-main-mattrpav-2

**Comment by gemmellr** [comment]
I'll be bowing out of the discussion now as I think this time would have been much better spent on other things, but to give a summary of my thoughts before I head as they doesnt really match what was just said.....

I think even 1/4 of that size (typed pre-edit, though still just about applies to the much smaller version there now) is too big personally, I dont think they need to have '..more' at all, I think they should link to the components download page with current releases, I dont think release-specific pages should be added for Artemis, and I dont think the entries need summary text but especially not counts needing manually composed for each release.

**Comment by mattrpav** [comment]
Styles to 'compact' the card

```
  <!-- News -->
  <div class="row align-middle">
    <div class="col-lg-4 col-md-auto">
      <div class="card card-orange" style="margin-bottom: 0">
        <div class="card-body" style="padding-top: .5rem;padding-left: .5rem;padding-bottom: .5rem;padding-right: .5rem;">
          <h6 class="card-title text-blue">ActiveMQ 5.16.3</h6>
          Bug fixes: 10, Dependency updates: 16, Enhancements: 3 <a href="{{site.baseurl}}/activemq-5016003-release" class="nav-link text-nowrap" style="display: unset; padding: unset">..more</a>
          <p class="text-right font-italic">August 18th, 2021</p>
        </div>
      </div>
    </div>
    <div class="col-lg-4 col-md-auto">
      <div class="card card-orange" style="margin-bottom: 0">
        <div class="card-body" style="padding-top: .5rem;padding-left: .5rem;padding-bottom: .5rem;padding-right: .5rem;">
          <h6 class="card-title text-blue">activemq-nms-amqp 1.8.2</h6>
          Bug fixes: 3, Dependency updates: 1, Enhancements: 2 <a href="{{site.baseurl}}/components/nms/" class="nav-link text-nowrap" style="display: unset; padding: unset">..more</a>
          <p class="text-right font-italic">August 16th, 2021</p>
        </div>
      </div>
    </div>
    <div class="col-lg-4 col-md-auto">
      <div class="card card-orange" style="margin-bottom: 0">
        <div class="card-body" style="padding-top: .5rem;padding-left: .5rem;padding-bottom: .5rem;padding-right: .5rem;">
          <h6 class="card-title text-blue">ActiveMQ Artemis 2.18.0</h6>
          Bug fixes: 79, Dependency updates: 19, Enhancements: 12 <a href="{{site.baseurl}}/components/artemis/download/release-notes-2.18.0" class="nav-link text-nowrap" style="display: unset; padding: unset">..more</a>
          <p class="text-right font-italic">August 16th, 2021</p>
        </div>
      </div>
    </div>
  </div>
```

**Comment by mattrpav** [comment]
Better sampling of news data to reflect not only releases

2021-activemq-main-mattrpav-3

**Comment by jbertram** [comment]
Here's a slightly modified version of my previous candidate:
Screenshot 2021-09-01 at 11-01-50 ActiveMQ
Changes:
 - Slightly larger text
 - smaller, text-based date in a different color
 - separator using less obtrusive color

**Comment by mattrpav** [comment]
>     * Slightly larger text
> 
>     * smaller, text-based date in a different color

Agreed, using 'Aug' and 'Sep' is better.  Plz share the html / css you used to generate

>     * separator using less obtrusive color

What is the grey color you used? I think we need a bit more space.. see my last render including non-release news item samples.

**Comment by lucastetreault** [comment]
image

Incorporated the feedback 👍

**Comment by jbertram** [comment]
I think the summary should be moved to be a tool-tip so it appears on hover in order to reduce the vertical real-estate _or_ move the whole thing as-is down below the first paragraph.

**Comment by mattrpav** [comment]
@lucastetreault can you incorporate these styles?

- [x] Gray, slimmer divider
- [x] Padding / margin fixes

```
 <!-- News -->
  <div class="row align-middle">
    <div class="col-lg-4 col-md-auto">
      <div class="card card-orange" style="margin-bottom: 0; border-left: 2px solid #6c757d !important">
        <div class="card-body" style="padding-top: .5rem;padding-left: .5rem;padding-bottom: .5rem;padding-right: .5rem;">
          <h6 class="card-title text-blue">ActiveMQ Camel</h6>
          activemq-camel has been migrated to Camel for 3.x. Migration guide available <a href="{{site.baseurl}}/activemq-5016003-release" class="nav-link text-nowrap" style="display: unset; padding: unset; margin-left: 0">..more</a>
          <p class="text-right font-italic" style="margin-bottom: 0">Aug 18, 2021</p>
        </div>
      </div>
    </div>
    <div class="col-lg-4 col-md-auto">
      <div class="card card-orange" style="margin-bottom: 0; border-left: 2px solid #6c757d !important">
        <div class="card-body" style="padding-top: .5rem;padding-left: .5rem;padding-bottom: .5rem;padding-right: .5rem;">
          <h6 class="card-title text-blue">CVE-314-14</h6>
          CVE-313-14 announced. Mitigation guide available <a href="{{site.baseurl}}/components/nms/" class="nav-link text-nowrap" style="display: unset; padding: unset; margin-left: 0">..more</a>
          <p class="text-right font-italic" style="margin-bottom: 0">Aug 16, 2021</p>
        </div>
      </div>
    </div>
    <div class="col-lg-4 col-md-auto">
      <div class="card card-orange" style="margin-bottom: 0; border-left: 2px solid #6c757d !important">
        <div class="card-body" style="padding-top: .5rem;padding-left: .5rem;padding-bottom: .5rem;padding-right: .5rem;">
          <h6 class="card-title text-blue">ActiveMQ Artemis 2.18.0</h6>
          Bug fixes: 79, Dependency updates: 19, Enhancements: 12 <a href="{{site.baseurl}}/components/artemis/download/release-notes-2.18.0" class="nav-link text-nowrap" style="display: unset; padding: unset; margin-left: 0">..more</a>
          <p class="text-right font-italic" style="margin-bottom: 0">Aug 16, 2021</p>
        </div>
      </div>
    </div>
  </div>
```

**Comment by mattrpav** [comment]
Summary from impromptu working session on Slack:

1. Move the news feed below first paragraph
2. Follow-on task to improve overall layout of front page

**Comment by gemmellr** [comment]
I'm kind of aghast this was merged already given some of the feedback points, and with all the scrappy fixup and revert commits. You can preview most changes locally.

I'll be removing the new individual download pages if someone doesn't beat me to it.

**Comment by mattrpav** [comment]
Hello @gemmellr-

The primary issue was mitigated in less than an hour after going live (the News page link). The other minor issue (grey box full length) is easily missed in a local preview, since was still displayed-- just not as wide as before.

@jbertram had reported an issue with vertical lines in the news feed cards not showing up, but that was a case of local browser caching and it cleared up on its own.

The release pages (active and past releases) are setup to be dynamic, as requested. The page-per-release was not intended to ruffle any feathers and is being addressed in a follow-on task being tracked here: [AMQ-8370](https://issues.apache.org/jira/browse/AMQ-8370)

@lucastetreault is a new contributor. The website is a great place to encourage involvement of people new to Apache and the Apache ActiveMQ community. I encourage you to consider giving him a few business hours to sort out the page-per-release changes.

I don't see any user-impacting issues at this time, if you see something-- I'd be happy to address it this evening.

**Pull request merged**